netplugin/plugin/netplugin_test.go

```diff
@@ -86,18 +115,19 @@ func TestNetPluginInitInvalidConfigMissingInstance(t *testing.T) {
 }
 
 func TestNetPluginInitInvalidConfigEmptyHostLabel(t *testing.T) {
+	// Test NetPlugin init failure when empty HostLabel provided
 	configStr := `{
-                    "drivers" : {
-                       "network": "ovs",
-                       "endpoint": "ovs",
-                       "state": "fakedriver"
-                    },
-                    "plugin-instance": {
-                       "host-label": "",
-					   "fwd-mode":"bridge",
-					   "db-url": "etcd://127.0.0.1:4001"
-                    }
-                  }`
+					"drivers" : {
```

Comment:
This is due to tabs before the chars, if we think spaces are better, i can switch it